integration-tests/tests/circuits.rs

```diff
@@ -58,20 +74,108 @@ async fn test_state_circuit_block(block_num: u64) {
     prover.verify().expect("state_circuit verification failed");
 }
 
+async fn test_tx_circuit_block(block_num: u64) {
+    const DEGREE: u32 = 20;
```

Comment:
Instead of doing this on each test, why don't we use a crate like: https://github.com/markhildreth/test-context to do test setup/teardown?